src/main/java/com/acoustic/salarycalculator/menu/SalaryCalculatorMenu.java

```diff
@@ -1,27 +1,27 @@
-package org.example.enumApproch.menu;
+package com.acoustic.salarycalculator.menu;
 
 import java.math.BigDecimal;
 import java.util.Arrays;
```

Comment:
2000 could be moved to const like MINIMUM_SALARY

Reply:
always try to not put some magic numbers in the code, reader very often doesn't know what it is and for what ;)